build/x86_64_amzn-2016.09/Dockerfile

```diff
@@ -1,8 +1,7 @@
 FROM amazonlinux:2016.09
 
 ADD http://ftp.tu-chemnitz.de/pub/linux/dag/redhat/el6/en/x86_64/rpmforge/RPMS/rpm-macros-rpmforge-0-6.el6.rf.noarch.rpm .
-RUN yum -y update \
```

Comment:
Just curious: Is `yum -y update` automatically triggered, or just not needed?

Reply:
Unlike `apt-get update`, which simply refreshes the repo contents, `yum update` upgrades all installed packages (similar to `apt-get upgrade`), so it was the wrong thing to do. The refresh of the repos (similar to `apt-get update`) happens automatically with `yum`.